src/SignInPage.tsx

```diff
@@ -72,11 +79,16 @@ const SignInPage: React.FunctionComponent<SignInPageProps> = ({isARCApp, usernam
         </Box>
       )}
       <Box className={classes.rightContainer} sx={{width: isARCApp ? '50%' : '100%'}}>
-        <AccountLogin
-          callbackFn={() => {}}
-          isArcSignIn={isARCApp}
-          usernameAndPasswordLogin={usernameAndPasswordLogin}
-        />
+        {(location.pathname === constants.publicPaths.SIGN_IN || location.pathname === '/') && (
+          <AccountLogin
+            callbackFn={() => {}}
+            isArcSignIn={isARCApp}
+            usernameAndPasswordLogin={usernameAndPasswordLogin}
+          />
+        )}
+        {featureToggles['USERNAME PASSWORD LOGIN'] && location.pathname === constants.publicPaths.RESET_PASSWORD && (
```

Comment:
Not very familiar with FeatureToggles in this app, but `'USERNAME PASSWORD LOGIN'` seems like a good candidate for a constant